furusiya/main_interface.py

```diff
@@ -33,13 +37,22 @@ def run(cls, to_run):
             panel=(SCREEN_WIDTH, PANEL_HEIGHT),
             fps_limit=LIMIT_FPS
         )
+        cls.saveload = SaveLoad(cls)
+
+        from model.key_binder import KeyBinder
+        cls.keybinder = KeyBinder(cls)
 
         to_run()
 
         print("Terminating ...")
 
         file_watcher.stop()
 
+    @classmethod
+    def set_keybinds_and_events(cls):
```

Comment:
Again, why not just set these on `Game`? Why pass in `cls`? To me, this should be a static method that operates on `Game.keybinder`.

Reply:
I would argue that this is actually less clear than operating directly on `Game`. But sure, whatever works for you.

Reply:
I checked my course notes, and this is apparently the way to go.